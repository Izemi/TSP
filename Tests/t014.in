Baltimore,39.17527778,-76.66833333
Istanbul,40.98222222,28.81277778
Seattle,47.45,-122.3116667
Columbus,39.99694444,-82.89222222
Boston,42.36305556,-71.00638889
Williamsport,41.24166667,-76.92194444
