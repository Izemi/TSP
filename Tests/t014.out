-given    :     23180.49 Baltimore Istanbul Seattle Columbus Boston Williamsport Baltimore
